modules/apps/commerce/commerce-order-content-web/src/main/resources/META-INF/resources/js/info_box/modal/InfoBoxModalTermInput.js

```diff
@@ -136,15 +145,15 @@ const InfoBoxModalTermInput = ({
 					)}
 				</>
 			)}
-			{hasTerms && !isOpen && (
+			{!!terms.length && !isOpen && (
```

Comment:
[NB]: This assumes that a selected term description exists, which is theoretically ok, and anyway falls back to an empty string. But there should be an actual validation and the condition should check for its actual existence as it's `dangerously set as inner HTML`. That thing could be configured to contain anything, even code. If there's at least 1 term, whatever is contained in `selectedTermDescription`, be it a string or anything else, it's gonna be put and run in the DOM.

Reply:
If it doesn't exist, it won't show anything. Why do we need a validation for an empty string?

Reply:
It will show something => an empty `div` (and potentially not just, but for rich-text sanitization admin-side we're already safe, so this is why this comment is non-blocking).